backend/webdav/webdav.go

```diff
@@ -482,6 +501,18 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 			rt: ntlmssp.Negotiator{RoundTripper: t},
 		}
 	}
+	if opt.Vendor == "other" {
+		// If available, prefer Digest authentication.
+		if checkForDigestAuth(opt.URL) {
+			fs.Debugf(nil, "using Digest authentication scheme")
+			client.Transport = &digest.Transport{
```

Comment:
Can this use rclone's transport? Not using rclone 's transport will cause some of rclone's features to stop working

Reply:
Fair enough :) I changed it accordingly and it seems to work just fine. I'll push it with my next commit.